src/lambda-wrapper.d.ts

```diff
@@ -0,0 +1,3 @@
+declare module 'lambda-expressless' {
+    declare function use(...handlers: any[]): void;
```

Comment:
Use method expects middleware functions, not `any`. And also return a function in style of lambda handler `(event: LambdaEvent, context: LambdaContext, callback: LambdaHandlerCallback): void`

Reply:
Made new commit. Can you please check and resolve if it is ok?